proxy/InkAPIInternal.h

```diff
@@ -338,6 +365,59 @@ class ConfigUpdateCbTable
   InkHashTable *cb_table;
 };
 
+class HttpHookState
+{
+public:
+  /// Scope tags for interacting with a live instance.
+  enum ScopeTag { GLOBAL, SESSION, TRANSANCTION };
```

Comment:
"TRANSANCTION"? Actually, change this to "GLOBAL, SSN, TXN" because that's more consistent with the rest of the code base.